src/Common/Doctrine/Type/UTCDateTimeType.php

```diff
@@ -64,17 +64,16 @@ public function convertToPHPValue($dateTimeString, AbstractPlatform $platform)
     /**
      * @return DateTimeZone
      */
-    private static function getUtc()
+    private static function getUtc(): DateTimeZone
     {
-        return self::$utc ? self::$utc : self::$utc = new DateTimeZone('UTC');
+        return self::$utc ?: self::$utc = new DateTimeZone('UTC');
```

Comment:
I know this is a correct notation, but I had to read it 5 times before a saw what is changed. Probably to confusing?